binding/dotnet/CobraTest/MainTest.cs

```diff
@@ -33,6 +34,7 @@ public class MainTest
         public static void ClassInitialize(TestContext _)
         {
             _accessKey = Environment.GetEnvironmentVariable("ACCESS_KEY");
+            _device = Environment.GetEnvironmentVariable("DEVICE") ?? "cpu:1";
```

Comment:
I think Ian commented on this in another PR, make sure to align the changes here with whatever gets decided in the other prs